Formula/digdag.rb

```diff
@@ -0,0 +1,21 @@
+class Digdag < Formula
+  desc "Workload Automation System"
+  homepage "https://github.com/treasure-data/digdag"
+  url "https://dl.digdag.io/digdag-0.8.1"
```

Comment:
Mind using `https://dl.digdag.io/digdag-0.8.1.jar` instead? That seems to be the full path, and helps make it a bit more obvious what is being downloaded.